core/src/com/tgf/twf/core/world/rules/Rules.java

```diff
@@ -2,25 +2,28 @@
 
 import com.tgf.twf.core.geo.Vector2;
 import com.tgf.twf.core.world.storage.Inventory;
-import com.tgf.twf.core.world.task.Action;
+import com.tgf.twf.core.world.storage.ResourceType;
+import com.tgf.twf.core.world.storage.SingleResourceInventory;
 
 import java.time.Duration;
 
 public class Rules {
+    public static final Inventory ONE_FOOD = SingleResourceInventory.of(ResourceType.FOOD, 1);
```

Comment:
This seems like the wrong place for this to live seeing as it isn't a rule. Can we define it on `SingleResourceInventory` and access it like `SingleResourceInventory.FOOD`?

Reply:
Ack, updating.